src/validation/profile.py

```
@@ -0,0 +1,45 @@
+import re
+from datetime import date
+from io import BytesIO
+
+from PIL import Image
+from fastapi import UploadFile
+
+from database.models.accounts import GenderEnum
+
+
+def validate_name(name: str):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)